docs/Course Wiki/ECE Course Offerings/ECE431.md

```diff
@@ -0,0 +1,75 @@
+
+## Course Structure
+
+This course has semi-weekly homeworks, two midterms, weekly prelabs and labs, a written lab report due every week, a final research project, and one final.
+
+The homeworks start weekly and tend to lighten up as the semester goes on. On exam weeks, the homework due date is usually extended. Homework sets are generally not too difficult and will primarily contain calculations, but there are some problems that test deeper understanding of the course content. Many of the homework problems are gone over in class before the assignment is due. Some homeworks will take under an hour, while others require more calculation and can be generally annoying. Students who pay attention and stay engaged in lecture will find these homeworks to be conceptually simple. 
+
+The midterms are in class (1 hour) and usually not very difficult with the averages being in the high 80's to low/mid 90's with small standard deviations. Very few practice exams are given out (1-3 per exam) and students. Cheatsheet policies may vary depending on the semester and the lead instructor. For the Spring 2023 semester, students were allowed to bring one double-sided 8.5x11 cheatsheet to the first exam, two double-sided 8.5x11 cheat sheets to the second exam, but only two double-sided 8.5x11 cheat sheets to the final exam. Students were highly recommended to reuse cheat sheets from prior exams, although this was not enforced.
+
+Overall, the course requires about 8-12 hours a week based on how difficult the lab is for the given week and also how much effort you put into the report.
+## Instructor
+
+Professor Haran usually teaches this course. Haran has an extensive background within electric machines as he is the head of the power group and oversees many research projects related to electric machinery. He has experience working as an engineer at General Electric. 
+
+## Course Tips
+
+The lab reports can be complicated based on the week and may need extra time to complete, so start early! Especially start the induction machine lab report early as it is one of the longest reports that you will have to write in the course. Professor Haran usually finishes the content early in the class period and expects students to ask questions, so do not be afraid to ask a question no matter how simple you may think it is. 
+The course can get overwhelming quickly as you keep learning about different types of electric machines at an increasing pace. Be sure that you understand the content because the exams (and homeworks) often ask you to explain what would happen if a parameter changed and why that would occur. 
```

Comment:
Add another space between this line and the previous line (so that markdown knows these are separate paragraphs).